i18n/en.json

```diff
@@ -8,6 +8,7 @@
 	"config-header-percona": "Percona XtraDB Cluster settings",
 	"config-percona-engine": "The only choice for now is InnoDB",
 	"config-percona-innodb": "This is your only choice",
-	"config-percona-engine-help": "What do you need help with?"
+	"config-percona-engine-help": "What do you need help with?",
+	"config-header-PerconaDB": "Percona DB extension"
```

Comment:
where is this used?

Reply:
Database option form header on the installer page
![image](https://user-images.githubusercontent.com/59472102/84161624-22828d00-aa78-11ea-84ef-f92f04d8ab55.png)

Reply:
But, that doesn't use the message text ("Percona DB extension"). It uses the mis-spelled "Percoda DB" from the getReadableName() function.